adventure.py

```diff
@@ -310,6 +462,15 @@ def player_move(motion):
     else:
         heartString = "O"
 
+    #damage_dealt_on_turn = False
+    for element in world.all_elements:
+        if element["character"] == stored_tile[0] and element["does_damage"] == True:
+            player.health -= element["damage"]
+            #damage_dealt_on_turn = True
+            if element["name"] == "spikes":
+                print(colorama.Fore.RED + "Spikes underfoot draw blood!" if with_colors else "Spikes underfoot draw blood!")
```

Comment:
I don't like how this prints __above__ the health string an in red. 
What I would like to see is this appended to some sort of "message string" that prints __after__ the health string. I would also prefer it in white. You could have the word "blood" in red though...
@Michael-N Thoughts on this?